internal/impl/tmpl/flake.nix.tmpl

```diff
@@ -0,0 +1,68 @@
+{
+    description = "A devbox shell";
+
+    inputs = {
+        # master branch can be slow because not all binaries are built and populated in the cache.
+        # using the nixpkgs-unstable tag ensures everything has been built and populated in the cache.
+        # https://www.reddit.com/r/NixOS/comments/ydr4po/comment/itv6oqo/?utm_source=reddit&utm_medium=web2x&context=3
+        nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
```

Comment:
Can we use the `nixpkgs` commit hash in the devbox.json? or is it supposed to download the latest unstable and ignore that commit hash?

Reply:
Ah, good point. Yes, I should use the nixpkgs-commit-hash from devbox.json

Longer term, I think we would deprecate the nixpkgs-commit-hash from devbox.json, and instead rely on the devbox.lock file (which will contain the commit-hash of nixpkgs as well)